### PR TITLE
fix(Build status): add all build statuses when calling rocket boost service

### PR DESCRIPTION
**Pull request by gal-yardeni**

### CODE REVIEW

File: keel-igor/src/main/kotlin/com/netflix/spinnaker/keel/services/ArtifactMetadataService.kt [hunk 1]
```diff
@@ -113,7 +113,7 @@ class ArtifactMetadataService(
     )
 
     return retry.executeSuspendFunction {
-      buildService.getArtifactMetadata(commitId = commitId.trim(), buildNumber = buildNumber.trim())
+      buildService.getArtifactMetadata(commitId = commitId.trim(), buildNumber = buildNumber.trim(), completionStatus = CompletionStatus.values().joinToString { it.name })
```
Comment (luispollo) [inline keel-igor/src/main/kotlin/com/netflix/spinnaker/keel/services/ArtifactMetadataService.kt:116]: I _think_ this works, but have you tested calling igor just to check? (I mean, as opposed to specifying the parameter as a list).

File: keel-igor/src/main/kotlin/com/netflix/spinnaker/model/Build.kt [hunk 2]
```diff
@@ -27,16 +27,26 @@ data class Build(
...
+  val timestamp: Instant? = null,
+  val remoteUrl: String? = null
 )
 enum class Result {
   SUCCESS, UNSTABLE, BUILDING, ABORTED, FAILURE, NOT_BUILT
 }
+
+enum class CompletionStatus {
+
+  INCOMPLETE,
```
[the remark this answers is not in the document]
Comment (gal-yardeni) [reply keel-igor/src/main/kotlin/com/netflix/spinnaker/model/Build.kt:46]: nope, its `INCOMPLETE` in boost :) (copied it from the current code in gate)

File: keel-igor/src/main/kotlin/com/netflix/spinnaker/keel/services/ArtifactMetadataService.kt (hunk 1 above)
Comment (luispollo) [reply keel-igor/src/main/kotlin/com/netflix/spinnaker/keel/services/ArtifactMetadataService.kt:116]: Never mind, it works.

Comment (gal-yardeni) [reply keel-igor/src/main/kotlin/com/netflix/spinnaker/keel/services/ArtifactMetadataService.kt:116]: I have tested it and it is working :) here's an example:
https://igor-prestaging.us-west-2.spinnaker.mgmt.netflix.net:7004/ci/builds?commitId=04dfb52b3d81b6a6c0f744b8e2cbc45f0cb40502&buildNumber=124&completionStatus=INCOMPLETE%2C%20SUCCEEDED%2C%20FAILED%2C%20ABORTED

**Review by luispollo — APPROVED** [review]
LGTM!

**Pull request merged**